etc/filter_sourcekitten_output.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+# use Exports.swift as the canonical list of re-exported symbols we need to document.
```

Comment:
the alternative would be to maintain a list of the symbols we mark :nodoc: in multiple places. this approach seemed less likely to lead to issues in the future